scene/2d/physics_body_2d.cpp

```diff
@@ -35,6 +35,9 @@
 #include "core/math/math_funcs.h"
 #include "core/method_bind_ext.gen.inc"
 #include "scene/scene_string_names.h"
+#include "core/list.h"
+#include "core/object.h"
+#include "core/rid.h"
```

Comment:
Includes should be listed in alphabetical order.

Reply:
Done!